src/common/file_entry.rs

```diff
@@ -49,6 +49,7 @@ impl FileEntry {
         Self::get_file_name(path)
             .filter(|file_name| !EXCLUDED_FILES.contains(file_name))
             .filter(|file_name| file_name.starts_with(pattern) || file_name.ends_with(pattern))
+            .filter(|file_name| !file_name.ends_with(".bak"))
```

Comment:
Please add a test for that case here https://github.com/dotenv-linter/dotenv-linter/blob/master/src/common/file_entry.rs#L108

Reply:
I also adjusted the backup test to see if the backup file was created with the right extension. (https://github.com/dotenv-linter/dotenv-linter/pull/367/files#diff-32743576dae795c7a900f15c203a6d54a537e73dbb3c7ea4a5771a7147b1eea9R25)